Added Cohere model support with test and setup lib to #660

fabian57fabian opened the pull request:
Added Cohere Model support as LLM for PR #660 

- Added **CohereModel** class in models in deepeval/models/cohere_model.py
- Added **test case** in tests/test_model_cohere.py
- Added cohere **package** installation in setup.py

penguine-ip commented:
Hey @fabian57fabian, due to the flexibility of our base LLM cohere support should go in the docs instead. Do you mind making a PR with the code you've written in the docs?

fabian57fabian commented:
> Hey @fabian57fabian, due to the flexibility of our base LLM cohere support should go in the docs instead. Do you mind making a PR with the code you've written in the docs?

I've written a small gist for this feature here: https://gist.github.com/fabian57fabian/5b4d67728277d3800e98995bef0bb764
In order to add it to your docs do i have to add another mdx file e.g. _models-use-cohere.mdx_ ?

fabian57fabian commented:
> Hey @fabian57fabian, due to the flexibility of our base LLM cohere support should go in the docs instead. Do you mind making a PR with the code you've written in the docs?

Reverted back and added models-cohere.mdx in docs in last commit.
